build/root/Makefile

```diff
@@ -213,7 +213,7 @@ test-e2e:
 	@echo "$$TEST_E2E_HELP_INFO"
 else
 test-e2e: ginkgo generated_files
-	go run hack/e2e.go -- -v --build --up --test --down
+	go run hack/e2e.go -- --build --up --test --down
```

Comment:
Btw - don't we want to have `--verbose-commands` instead?

Reply:
that's default now

Reply:
that was done in https://github.com/kubernetes/test-infra/pull/5452

Reply:
I see.. thanks